packages/button/src/ButtonBase.tsx

```diff
@@ -56,19 +57,63 @@ function getVariantCss(props: ButtonProps): string {
      */
     case 'primary':
       return `
-        background-color: ${shadeBlack};
-        color: ${shadeWhite};
+        background-color: ${props.disabled ? grey20() : shadeBlack};
+        color: ${props.disabled ? grey40() : shadeWhite};
+        disabled: ${props.disabled}
         &:focus {
           outline: 0.2rem solid ${informative50()};
           outline-offset: 0.2rem;
         }
+        ${
+          props.disabled
+            ? `
+        &:hover {
+          background-color: ${grey20()};
+          cursor: default;
+        }
```

Comment:
Can we improve the indentation for readability?

```ts
`
  &:hover {
    background-color: ${grey20()};
    cursor: default;
  }
`
```

Reply:
The linter is complaining when i try to do anything but the current setup - thoughts?

Reply:
Ah ignore me then